java/com/jfreechartextensions/axistransformations/ExtendedNumberAxis.java

```diff
@@ -34,8 +35,25 @@ public void setTransform(BasicAxisTransform transform) {
         this.transform = transform == null ? newNullAxisTransform() : transform;
     }
 
+    public BasicAxisTransform getTransform() {
+        return transform;
+    }
+
+    public Range getDataRange() {
+        if (dataRange == null) {
```

Comment:
Do we always want to return the existing range?  What happens if the axis gets autoranged?  Will we be using the right range?

Reply:
dataRange is getting updated in autoAdjustRange() function also.